draft-jholland-quic-multicast.md

```diff
@@ -452,7 +453,7 @@ MC_CHANNEL_ACK Frame {
 
 ## MC_PATH_RESPONSE {#path-response-frame}
 
-MC_PATH_RESPONSE frames are sent from client to server in a unicast connection in response to an PATH_CHALLENGE received in a channel.  Like PATH_RESPONSE but includes a channel id.
+MC_PATH_RESPONSE frames are sent from a client to a server in a unicast connection in response to a PATH_CHALLENGE received in a channel.  Like PATH_RESPONSE but includes a channel id.
```

Comment:
This should also be "the unicast connection" instead of "a unicast connection", right?